frontend/src/pages/Calendars/Calendars.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import PageTemplate from '../../templates/PageTemplate/PageTemplate';
+import CalendarList from '../../components/CalendarList/CalendarList';
+import { constants } from '../../assets/configs/constants';
+
+const Calendars = () => {
+   const [ calendars, setCalendars ] = useState( [] );
+   const [ isLoading, setIsLoading ] = useState( true );
+   useEffect( () => {
+      const controller = new AbortController();
```

Comment:
Where does `AbortController` come from and what does it do? I can't see it among imports.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/AbortController